assignments/objects.js

```diff
@@ -33,13 +63,30 @@ const example = {
 
 // Antonietta's Gender
 
+console.log(intern1.name);
+console.log(intern2.id);
+console.log(intern3.email);
+console.log(intern4.name);
+console.log(intern5.gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+intern2.speak = function () {
+  console.log("Hello, my name is Kennan!");
+}
+console.log(intern2.speak());
```

Comment:
This is actually logging out undefined after it logs out the `console.log()` inside of the `intern2.speak()` method. That is because you are logging the return value of the `intern2.speak()` method, but you aren't returning anything in that function. 

How I would have done it:
```
intern2.speak = function () {
  return "Hello, my name is Kennan!";
}
```